### PR TITLE
Fixes BOs

### PR DESCRIPTION
**Pull request by ike709**
Tested and working.

### CODE REVIEW

File: code/modules/jobs/job_types/bridge.dm [hunk 1]
```diff
@@ -16,10 +16,10 @@ Bridge Officer
 
 	outfit = /datum/outfit/job/bofficer
 
-	access = list(access_heads, access_helm, access_maint_tunnels)
-	minimal_access = list(access_heads, access_helm, access_maint_tunnels)
+	access = list(GLOB.access_heads, GLOB.access_helm, GLOB.access_maint_tunnels)
+	minimal_access = list(GLOB.access_heads, GLOB.access_helm, GLOB.access_maint_tunnels)
 
-GLOBAL_LIST_INIT(posts = list("weapons", "helms"))
+var/list/posts = list("weapons", "helms")
```
Comment (ike709) [inline code/modules/jobs/job_types/bridge.dm]: Posts is only used in this file. No reason to make it a global list.

File: code/__DEFINES/jobs.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 #define ROBOTICIST		(1<<8)
 #define AI_JF				(1<<9)
 #define CYBORG			(1<<10)
+#define BOFFICER  (1<<11)
```
Comment (ike709) [inline code/__DEFINES/jobs.dm:15]: I don't know what these numbers mean but they're a pre-rebase value.

File: ftl13.dme [hunk 3]
```diff
@@ -1438,6 +1438,7 @@
 #include "code\modules\jobs\access.dm"
 #include "code\modules\jobs\jobs.dm"
 #include "code\modules\jobs\job_types\assistant.dm"
+#include "code\modules\jobs\job_types\bridge.dm"
```
Comment (ike709) [inline ftl13.dme:1441]: w e w l a d

File: code/modules/jobs/job_types/bridge.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/modules/jobs/job_types/bridge.dm]: it's a global list either way, making it use GLOB instead allows admins to look at it in game

**Pull request merged**